src/sass/_common/base/constants.scss

```diff
@@ -38,6 +37,8 @@ $COLOR_LIGHT_GRAY    : $fillColor;     // #f2f2f2
 $COLOR_GRAY          : $borderColor;   // #dedede
 $COLOR_DARK_GRAY     : #44484c;
 $COLOR_GREEN         : $positiveColor; // #2e8836
+$COLOR_BORDER_GRAY   : #f2f3f4;
+$COLOR_BALLOON_GRAY  : #d6dadb;
```

Comment:
I think these two colours aren't being used anywhere

Reply:
I've removed any missed usage in my chart control PR https://github.com/deriv-com/binary-static/pull/19

Reply:
I've removed these.